pkl/evaluator_manager_exec.go

```diff
@@ -206,5 +206,26 @@ func (e *execEvaluator) deinit() error {
 	close(e.out)
 	close(e.closed)
 	// TODO: graceful shutdown
-	return e.cmd.Process.Kill()
+	if err := e.cmd.Process.Signal(os.Interrupt); err != nil {
+		internal.Debug("Failed to interrupt process: %v", err)
```

Comment:
Does this `err` always contain the PID? If not, I think it might be useful to make sure the PID is included in the error message. (Here and below)